marketplace_builder/views/pages/tutorials/custom-model-types/custom-model-types.liquid

```diff
@@ -7,7 +7,16 @@ slug: tutorials/custom-model-types/custom-model-types
 searchable: true
 ---
 
-**Custom Model Types** have multiple use cases:
+## What is Custom Model Type?
+
+**Custom Model Type is an object that describes all Customization objects that belongs to it.**
```

Comment:
How about adding "You can think of it as of DB table, whereas Customizations are records inside this table - hence Customization must always be associated with exactly one CustomModelType" ?

Reply:
I've added your example and @andrei example, @diana-lakatos can you review please?